svensk-exegetisk-arsbok.csl

```diff
@@ -0,0 +1,782 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only" default-locale="sv-SE">
+  <info>
+    <title>Svensk Exegetisk Årsbok</title>
```

Comment:
Should the journal title be in title case (which is mostly an English habit)? Maybe "Svensk exegetisk årsbok" is better?